### DISCUSSION TITLE
Pull Request - Update to Feature Request Issues form

### DISCUSSION THREAD

**Discussion by TFWol** [open-webui/open-webui#7906] [Ideas]
Posting here per the *Pull Request Checklist* to get permission.

Since you're mostly focused on more important things like coding, I thought I'd help by revamping the Feature Request form to guide users to read and provide better input.

What it looks like when they click Feature Request:
![image](https://github.com/user-attachments/assets/f170158a-5694-47dd-ad11-c9008da4f82d)

**Comment by pktiuk** [comment]
The requirement of creating discussion before proposing even a minor change really slows down development. :(

**Reply by TFWol** [reply]
No worries there; it's only needed if it's the first time you've done a request in the repo 😃 

[This](https://github.com/open-webui/open-webui/blob/main/.github/pull_request_template.md) pops up when you do one.

**Comment by TFWol** [comment]
Never mind, looks like Issues tab is gone and everything's been chucked into Discussions ☹️

**Comment by TFWol** [comment]
The Issues tab reappeared. I'm going to just submit pull request and point at this discussion.

**Comment by TFWol** [comment]
Merged 😃
